test-zeekoe.py

```diff
@@ -244,6 +267,7 @@ def main():
 
     elif args.command == SCENARIO:
         info("Running scenario: %s" % ', '.join(command_list))
+        check_blockchain_maturity(url)
```

Comment:
Add a check to see if in sandbox mode first.

Reply:
done! In the same commit also changed the default argument from `"sandbox"` to `SANDBOX` since that's what I think it's supposed to be.